### ISSUE TITLE
Error migrating

### ISSUE THREAD

**Issue by yonaskolb** [workshop/struct#116]
## New Issue Checklist

- [x] Updated struct to the latest version
- [x] I have read the [documentation](https://github.com/workshop/struct/wiki) to make sure my issue isn't covered already

## Struct version
[2.1.0]

## Context

Migrating an Xcode project with Struct 2.1.0 fails with error:
```
Error: Invalid spec object. Project version 2.1.0 is unsupported by this version of struct.
```
This was also a similar problem with the initial 2.0.0 release, though there the generated spec was an old version. I guess a test checking migration would be good

**Comment by lyptt** [comment]
Hi @yonaskolb, thanks for filing this issue! It appears to be fairly easy to reproduce, so I'll get this fixed up shortly.

**Comment by lyptt** [comment]
This is now fixed in version 2.1.1.
